app/views/host/_config.html.haml

```diff
@@ -45,22 +45,20 @@
             %p.form-control-static
               = h(item[:description])
 - when "network"
-  %table{:style => "height: 100px;"}
-    %tr
-      %td{:width => "55%", :valign => 'top'}
-        -# Create divs for each VM to display as the mouse hovers over each VM node
-        - @tree_vms.each do |v|
-          #outer
-            #inner
-              %div{:id => "v-#{v.id}", :style => "display: none; z-index: 10; width: 72px; height: 72px; padding: 0 0 0 0;"}
-                %div{:style => "margin-left: -78px;"}
-                  = render_quadicon(v, :mode => :icon, :size => 72, :typ => :listnav)
-                  %div
-                    %center{:style => "color: #000;"}
-                      = h(v.product_name)
-                      %br
-                      = h(v.service_pack)
-        = render :partial => "network_tree"
+  .row
+    .col-md-10
+      = render :partial => "network_tree"
+    .col-md-2.hidden-sm.hidden-xs
```

Comment:
:smiley: i know :smiley: but is it okay to use them in this context? I mean hiding the hover-quadicons in small screens where you usually have no space and no hover (touchscreen)?